sdk/data_cosmos/src/operations/list_collections.rs

```diff
@@ -116,7 +116,7 @@ impl ListCollectionsResponse {
 }
 
 impl Continuable for ListCollectionsResponse {
-    fn continuation(&self) -> Option<String> {
-        self.continuation_token.clone()
+    fn continuation(&self) -> Option<Continuation> {
+        self.continuation_token.clone().map(|x| x.into())
```

Comment:
This might be ever so clearer:
```suggestion
        self.continuation_token.clone().map(Continuation::from)
```